src/template/index.js

```diff
@@ -201,9 +199,11 @@ async function openwhiskAdapter(params = {}) {
         region: process.env.__OW_REGION,
       },
       func: {
-        name: names.join('/'),
-        version: process.env.__OW_ACTION_VERSION,
-        app: namespace,
+        name,
+        version,
+        package: packageName,
+        app: process.env.__OW_NAMESPACE,
```

Comment:
btw: there is a difference between the namespace in the `__OW_ACTION_NAME` and the `__OW_NAMESPACE`.
the first is the one where the function is deployed, and the later is the one where it is executed.
I think the latter is more important. the first can still be retrieved using the `fqn`